src/serverless-openapi-typescript.ts

```diff
@@ -216,6 +231,8 @@ export default class ServerlessOpenapiTypeScript {
                 } else {
                     method.tags = [tagName];
                 }
+
+                method.operationId = `${this.serverless.service.service}-${method.operationId}`;
```

Comment:
This field is also responsible for the url also - for example, `https://docs.env0.com/reference/serverless-openapi-typescript-demo-updatefunc`